client/app/components/faq/FaqList.js

```diff
@@ -133,11 +134,15 @@ const mapDispatchToProps = dispatch => {
 
 FaqList.propTypes = {
 	onFAQLoaded: PropTypes.func.isRequired,
-	currentQuestions: PropTypes.array
+	navigation: PropTypes.object.isRequired,
+	currentQuestions: PropTypes.arrayOf(PropTypes.string),
+	navigate: PropTypes.func
 };
 
 FaqList.defaultProps = {
-	currentQuestions: []
+	currentQuestions: [],
+	navigate:() => {}
+
```

Comment:
Empty space?

Reply:
@pratyushpal please fix this in your next ticket.